app/views/overrides/organizations/_edit_override.html.erb

```diff
@@ -8,11 +8,19 @@
     <%= selectable_f(f, :service_level, options_for_select(service_level_options, service_level_selected), {}, :label => _('Default System SLA')) %>
   <% end %>
 
+
   <%= field(f, _('Debug Certificate'),
             :help_inline => _('This certificate allows a user to view the repositories in any environment from a browser.')) do
     button_tag _('Generate and Download'), :id => :download_debug_cert_key,
                :type => :button, :style => 'margin-top: 8px;', :class => 'btn btn-small',
                'data-url' => "/katello/api/v2/organizations/#{@taxonomy.id}/download_debug_certificate"
   end %>
 
-<% end%>
+  <%= field(f, _('Simple Content Access'),
```

Comment:
The toggle is showing up for orgs that don't have a manifest imported - that should be prevented